File: ios/Objective-C/HelloWorld/HelloWorld/ResultViewController.m
//
//  ResultViewController.m
//  HelloWorld
//
//  Created by admin on 2022/2/25.
//  Copyright © 2022 dynamsoft. All rights reserved.
//

#import "ResultViewController.h"
#import "DDNDataManager.h"

@interface ResultViewController ()

@end

@implementation ResultViewController

@synthesize imageView;

- (void)viewDidLoad {
    [super viewDidLoad];
    [self configResultView];
    // Do any additional setup after loading the view.
}

- (void)configResultView{
    self.view.backgroundColor = [UIColor blackColor];
    CGFloat w = [[UIScreen mainScreen] bounds].size.width;
    CGFloat h = [[UIScreen mainScreen] bounds].size.height;
    
    imageView = [[UIImageView alloc] initWithFrame:CGRectMake(0, 0, w, h)];
    imageView.userInteractionEnabled = YES;
    imageView.contentMode = UIViewContentModeScaleAspectFit;
    [imageView setImage:[DDNDataManager instance].resultImage];
    [self.view addSubview:self->imageView];
}

@end

File: ios/Swift/HelloWorld/HelloWorld/ResultViewController.swift
//
//  ResultViewController.swift
//  HelloWorld
//
//  Copyright © Dynamsoft. All rights reserved.
//

import UIKit

class ResultViewController : UIViewController {
    private var imageView: UIImageView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        configResultView()
    }

    func configResultView() {
        view.backgroundColor = UIColor.black
        let w = UIScreen.main.bounds.size.width
        let h = UIScreen.main.bounds.size.height

        imageView = UIImageView(frame: CGRect(x: 0, y: 0, width: w, height: h))
        imageView.isUserInteractionEnabled = true
        imageView.contentMode = .scaleAspectFit
        imageView.image = DDNDataManager.instance.resultImage
        view.addSubview(imageView)
    }
}
